libdrgn/dwarf_index.h

```diff
@@ -424,6 +492,32 @@ struct drgn_error *
 drgn_dwarf_index_iterator_next(struct drgn_dwarf_index_iterator *it,
 			       Dwarf_Die *die_ret, uint64_t *bias_ret);
 
+
+/**
+ * Get the next matching drgn_dwarf_die from a DWARF index iterator.
+ *
+ * This is a more bare-bones version of drgn_dwarf_index_iterator_next, not doing the conversion to a Dwarf DIE.
+ *
+ * @param[in] it DWARF index iterator.
+ */
+struct drgn_dwarf_index_die *
+drgn_dwarf_index_iterator_next_die(struct drgn_dwarf_index_iterator *it);
```

Comment:
I'd rather get rid of the old `drgn_dwarf_index_iterator_next()` and make everywhere do the conversion manually.